client/components/cart.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getCartFIXME } from '../store';
+import { Link } from 'react-router-dom';
+
+class Cart extends Component {
+  componentDidMount() {
+    // this.props.getCartFIXME();
+  }
+  render() {
+    // const cart  = this.props.cart;
+    // const boats = cart.boats
+
+    const initialBoats = [
+      {
+        id: 1,
+        name: 'Pearl',
+        imageUrl: null,
+        description: '',
+        cost: 815750,
+      },
+      {
+        id: 3,
+        name: 'Forever Young',
+        imageUrl: null,
+        description: '',
```

Comment:
@JoshuaSkootsky why does the cart component render boats?

Reply:
The shopping cart contains boats.

Our imagined use case is that the User selects boats to add to the shopping cart, and can then view the boats in the shopping cart through the Cart React component.